ocaml/xapi/xapi_host.ml

```diff
@@ -885,7 +885,7 @@ let set_ssl_legacy ~__context ~self ~value =
 	let old = Db.Host.get_ssl_legacy ~__context ~self in
 	Db.Host.set_ssl_legacy ~__context ~self ~value;
 	if old <> value then (
```

Comment:
Query: are these functions thread-safe. Particularly the one that would do a `service ... restart`? I suppose they're only called at Xapi startup so maybe they can't be called more than once. If they're blocking I suppose we're OK.